### DISCUSSION TITLE
Filter on type support

### DISCUSSION THREAD

**Discussion by sanchewy** [BurntSushi/ripgrep#2163] [Q&A] [answered]
For the below command
```
rga -i -t pdf "FooBar"
```
`-t` type works for me when I use `pdf`, but for `docx` I get
```
rga -i -t docx "Software"
unrecognized file type: docx
```
I have the poppler and pandoc adapters installed.

**Comment by sanchewy** [comment]
Using globs does work, however.
```
rga -i "Software" -g '*.docx'
```

**Comment by BurntSushi** [comment, marked as answer]
This is the wrong repo.

**Reply by sanchewy** [reply]
Oh crap sorry. Thanks.
